<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDlg</class>
 <widget class="QDialog" name="MainDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PC-BSD User Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pcbsdusermanager.qrc">
    <normaloff>:/add_group.png</normaloff>:/add_group.png</iconset>
  </property>
  <layout class="QGridLayout">
   <item row="1" column="0">
    <widget class="QTabWidget" name="mainTab">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="userTab">
      <attribute name="title">
       <string>&amp;Users</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="2" column="0" colspan="2">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QPushButton" name="newButton">
           <property name="text">
            <string>&amp;Add</string>
           </property>
           <property name="icon">
            <iconset resource="pcbsdusermanager.qrc">
             <normaloff>:/add_user.png</normaloff>:/add_user.png</iconset>
           </property>
           <property name="shortcut">
            <string>Alt+A</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteButton">
           <property name="text">
            <string>&amp;Remove</string>
           </property>
           <property name="icon">
            <iconset resource="pcbsdusermanager.qrc">
             <normaloff>:/delete_user.png</normaloff>:/delete_user.png</iconset>
           </property>
           <property name="shortcut">
            <string>Alt+R</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>231</width>
             <height>30</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QListWidget" name="userList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="uidLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>User ID:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="uidBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="fullnameLabel">
            <property name="text">
             <string>Full Name:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="fullnameBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="homeLabel">
            <property name="text">
             <string>Home Directory:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="homeBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="shellLabel">
            <property name="text">
             <string>Shell:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="shellBox">
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="autoCompletion">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="textLabel1">
            <property name="text">
             <string>Primary Group:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="groupBox"/>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="passwordButton">
            <property name="text">
             <string>&amp;Change Password...</string>
            </property>
            <property name="icon">
             <iconset resource="pcbsdusermanager.qrc">
              <normaloff>:/change_password.png</normaloff>:/change_password.png</iconset>
            </property>
            <property name="shortcut">
             <string>Alt+G</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="2">
           <widget class="QGroupBox" name="group_personacrypt">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="title">
             <string>PersonaCrypt</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="leftMargin">
              <number>1</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>1</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QFrame" name="frame_2">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <property name="leftMargin">
                 <number>2</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>2</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QToolButton" name="tool_PCKey_export">
                  <property name="text">
                   <string>Export Key</string>
                  </property>
                  <property name="icon">
                   <iconset resource="pcbsdusermanager.qrc">
                    <normaloff>:/leftarrow.png</normaloff>:/leftarrow.png</iconset>
                  </property>
                  <property name="toolButtonStyle">
                   <enum>Qt::ToolButtonTextBesideIcon</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="tool_PCKey_import">
                  <property name="text">
                   <string>Import Key</string>
                  </property>
                  <property name="icon">
                   <iconset resource="pcbsdusermanager.qrc">
                    <normaloff>:/rightarrow.png</normaloff>:/rightarrow.png</iconset>
                  </property>
                  <property name="toolButtonStyle">
                   <enum>Qt::ToolButtonTextBesideIcon</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_3">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <property name="leftMargin">
                 <number>2</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>2</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QToolButton" name="tool_PCKey_disable">
                  <property name="text">
                   <string>Disable Key (No Data)</string>
                  </property>
                  <property name="icon">
                   <iconset resource="pcbsdusermanager.qrc">
                    <normaloff>:/delete_user.png</normaloff>:/delete_user.png</iconset>
                  </property>
                  <property name="toolButtonStyle">
                   <enum>Qt::ToolButtonTextBesideIcon</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="tool_PCKey_disablecopy">
                  <property name="text">
                   <string>Disable Key (Import Data)</string>
                  </property>
                  <property name="icon">
                   <iconset resource="pcbsdusermanager.qrc">
                    <normaloff>:/delete_user.png</normaloff>:/delete_user.png</iconset>
                  </property>
                  <property name="toolButtonStyle">
                   <enum>Qt::ToolButtonTextBesideIcon</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="groupTab">
      <attribute name="title">
       <string>&amp;Groups</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="7" column="0" colspan="7">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QPushButton" name="addGroupBut">
           <property name="text">
            <string>A&amp;dd</string>
           </property>
           <property name="icon">
            <iconset resource="pcbsdusermanager.qrc">
             <normaloff>:/add_group.png</normaloff>:/add_group.png</iconset>
           </property>
           <property name="shortcut">
            <string>Alt+D</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeGroupBut">
           <property name="text">
            <string>&amp;Remove</string>
           </property>
           <property name="icon">
            <iconset resource="pcbsdusermanager.qrc">
             <normaloff>:/delete_group.png</normaloff>:/delete_group.png</iconset>
           </property>
           <property name="shortcut">
            <string>Alt+R</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>140</width>
             <height>31</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="availLbl">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Available:  </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3" colspan="2">
        <spacer name="spacer23">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>91</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="makeMemberBut">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="pcbsdusermanager.qrc">
           <normaloff>:/rightarrow.png</normaloff>:/rightarrow.png</iconset>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="makeAvailableBut">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="pcbsdusermanager.qrc">
           <normaloff>:/leftarrow.png</normaloff>:/leftarrow.png</iconset>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <spacer name="spacer4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>31</width>
           <height>110</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3">
        <spacer name="spacer3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>30</width>
           <height>110</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="5">
        <widget class="QLabel" name="groupLbl">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Members:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="6" rowspan="5">
        <spacer name="spacer24">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>50</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2" rowspan="4">
        <widget class="QListWidget" name="availableList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="5">
        <widget class="QListWidget" name="groupList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="5" rowspan="4">
        <widget class="QListWidget" name="memberList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="headerLayout1">
     <item>
      <layout class="QHBoxLayout" name="headerLayout2">
       <item>
        <widget class="QLabel" name="textLabel1_2">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>PC-BSD User Manager - Advanced Mode</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer12">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>219</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="simpleBut">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>&amp;Simple View</string>
         </property>
         <property name="shortcut">
          <string>Alt+V</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line3">
       <property name="frameShape">
        <enum>QFrame::HLine</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>simpleBut</tabstop>
  <tabstop>mainTab</tabstop>
  <tabstop>newButton</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>makeMemberBut</tabstop>
  <tabstop>makeAvailableBut</tabstop>
  <tabstop>addGroupBut</tabstop>
  <tabstop>removeGroupBut</tabstop>
 </tabstops>
 <resources>
  <include location="pcbsdusermanager.qrc"/>
 </resources>
 <connections/>
</ui>
